### PR TITLE
Adds missing input and change events on radio and checkboxes

### PR DESCRIPTION
**Pull request by zonzujiro**
Fixing https://github.com/wix-incubator/unidriver/issues/90

### CODE REVIEW

**Comment by GabiGrin** [comment]
thanks @zonzujiro! LGTM!
PR build fail looks unrelated

@borisd9 @mosheKerbel @shlomitc any objections to merge this?

**Comment by zonzujiro** [comment]
@GabiGrin as far as I understand, we can merge it? But I can't because of failed builds.

File: adapters/jsdom-react/src/index.ts [hunk 1]
```diff
@@ -86,6 +94,12 @@ export const jsdomReactUniDriver = (containerOrFn: ElementOrElementFinder): UniD
 		}
 	};
 
+	const handleCheckableInput = async (input: HTMLInputElement) => {
```
Comment (mosheKerbel) [inline adapters/jsdom-react/src/index.ts]: you either miss an `await` on the caller, or this `async` is redundant.

Comment (mosheKerbel) [reply adapters/jsdom-react/src/index.ts]: besides that we are good in terms of WSR, all tests pass.

**Pull request merged**